src/ResourceManager/Network/Commands.Network/packages.config

```diff
@@ -2,5 +2,5 @@
 <packages>
   <package id="AutoMapper" version="6.0.2" targetFramework="net452" />
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Network" version="19.3.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="19.4.0-preview" targetFramework="net452" />
```

Comment:
Do you still need this?

Reply:
Yes - these changes are present in the official 19.4 SDK.